assets/demo-todo-app.spec.js

```diff
@@ -285,21 +285,21 @@ test.describe('Clear completed button', () => {
 
   test('should display the correct text', async ({ page }) => {
     await page.locator('.todo-list li .toggle').first().check();
-    await expect(page.locator('.clear-completed')).toHaveText('Clear completed');
+    await expect(page.getByRole('button', { name: 'Clear completed' })).toHaveText('Clear completed');
```

Comment:
this assertion makes no sense. Say the following instead:
await expect(page.getByRole('button', { name: 'Clear completed' })).toBeVisible();

Reply:
ah yes, you are right